l3kernel/l3fp-symbolic.dtx

```diff
@@ -445,23 +402,23 @@
   }
 \cs_new:Npn \@@_symbolic_unary_to_tl:NNw #1#2 , #3 @
   {
-    \use:ff
+    \use:ee
       { \@@_symbolic_cs_arg_to_fn:NN #1#2 }
       { \exp_last_unbraced:Nf ( { \@@_to_tl_dispatch:w #3 } ) }
   }
 \cs_new:Npn \@@_symbolic_binary_to_tl:Nww #1, #2; #3; @
   {
-    \use:ff
+    \use:ee
       { \exp_last_unbraced:Nf ( { \@@_to_tl_dispatch:w #2; } ) }
       {
-        \use:ff
+        \use:ee
           { \@@_types_cs_to_op:N #1 }
           { \exp_last_unbraced:Nf ( { \@@_to_tl_dispatch:w #3; } ) }
```

Comment:
```suggestion
        \use:e
          {
            \@@_types_cs_to_op:N #1
            ( \@@_to_tl_dispatch:w #3; )
          }
```

Reply:
Also, nesting `\use:e(e)` inside `\use:e(e)` seems unnecessary here.

Reply:
Good point - I've sorted that on main as it also applies there